src/backend/aspen/api/schemas/base.py

```diff
@@ -4,7 +4,12 @@
 
 
 class BaseRequest(BaseModel):
-    pass
+    class Config:
+        """Extra configuration options"""
+
+        anystr_strip_whitespace = (
+            True  # remove leading/trailing whitespace from strings
```

Comment:
Why is this in parens? Is it just an auto-formatter thing? It wanted to stay under 80chars or something? My first thought was there was some kind of tuple thing going on for the config, but I realize that's not the case now.

Reply:
the formatting tool didn't like my comment making the line super long so it wrapped the value onto another line with parens.